lib/src/lib/table/components/table-cell/table-cell.component.ts

```diff
@@ -131,19 +135,16 @@ export class Lab900TableCellComponent<T = any> implements OnDestroy {
   public readonly cellHeaderClass$: Observable<string>;
   public readonly sticky$: Observable<boolean>;
   public readonly cellFooter$: Observable<string>;
-  public readonly sort$: Observable<Lab900Sort[] | null>;
+  public readonly sort$: Observable<Lab900Sort[] | null> =
+    this.tableService.sort$;
 
   public readonly defaultCellRenderer = DefaultCellRendererComponent;
   public readonly defaultHeaderRenderer = DefaultColumnHeaderRendererComponent;
 
   @Input({ required: true })
   public trackByTableFn!: TrackByFunction<T>;
 
-  public constructor(
-    @Optional() @SkipSelf() public table: MatTable<any>,
```

Comment:
what did @skipself do? and did you account for that?

Reply:
This was for the compiler, it complained about circular dependency risk but this didn't make sense. It doesn't complain anymore with inject()